transcoders.go

```diff
@@ -373,3 +374,14 @@ func dnsStB(s string) ([]byte, error) {
 func dnsBtS(b []byte) (string, error) {
 	return string(b), nil
 }
+
+var TranscoderCertHash = NewTranscoderFromFunctions(certHashStB, certHashBtS, nil)
+
+func certHashStB(s string) ([]byte, error) {
+	_, data, err := multibase.Decode(s)
```

Comment:
Are there any other requirements here? e.g. is it a multihash, is it just 32 bytes, etc.

Reply:
I added the validation on encoding.